<ui version="4.0" >
 <class>PasswordsDialog</class>
 <widget class="QDialog" name="PasswordsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Passwords</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <spacer name="spacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" >
    <widget class="SearchLineEdit" name="search" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="removeButton" >
       <property name="text" >
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="detailsButton" >
       <property name="text" >
        <string>&amp;Details</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="EditTableView" name="table" >
     <property name="tabKeyNavigation" >
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode" >
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="horizontalScrollMode" >
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid" >
      <bool>false</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>There is no password storage backend yet. Maybe you want to write one :)</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchLineEdit</class>
   <extends>QLineEdit</extends>
   <header>searchlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>EditTableView</class>
   <extends>QTableView</extends>
   <header>edittableview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>search</tabstop>
  <tabstop>table</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PasswordsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>446</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel" >
     <x>257</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
